test1/stylecontact.css

```diff
@@ -0,0 +1,185 @@
+*{
+    margin: 0;
+    padding: 0;
+}
+@font-face{
+    font-family: GTZ;
+    src: url(img/font/GTZirkon-Book.woff2);
+}
+
+body{
+    font-family: GTZ;
+}
+.nav-main{
+    background-color: #ffe8dc;
+    height: 700px;
+}
+.nav-top{
+    display: flex;
+}
+.nav-name p{
+    font-size: 60px;
+    margin-top: 30px;
+    margin-left: 50px;;
+}
+.nav-menu p{
+    font-size: 35px;
+    margin-left: 400px;
+    margin-top: 30px;
+
+}
+.nav-bar{
+    display: none;
+
+}
+.nav-main a{
+    text-decoration: none;
+    color: #2b2b2b;
+}
+.nav-contact{
+    margin-top:-200px ;
+    margin-left: 50px;
+}
+.nav-contact a{
+    font-size: 65px;
+    text-decoration: none;
+    color:#2b2b2b;
+}
+.nav-contact p{
+    font-size: 35px;
+}
+.email::after{
+    content: "*";
+    color: red;
+}
+.message::after{
+    content: "*";
+    color: red;
+}
+.detail label{
+    font-size: 30px;
+}
+input{
+    width: 50%;
+    background-color: rgba(171, 201, 209, 0.486);
+    border: 1px solid rgba(171,201,209,0.486);
+    height: 50px
+}
+textarea{
```

Comment:
Don't directly style HTML elements like this. Always provide a class name or scope it under a class name and style.